src/styles/blocks/page.scss

```diff
@@ -0,0 +1,27 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
it's better to reset such things for list of tags you really need
keep only `box-sizing` in `*` selector